src/Configuration/ValueObject/Serializer/DatabaseConfigSerializer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\ValueObject\Serializer;
+
+use Keboola\DbExtractorConfig\Configuration\ValueObject\DatabaseConfig;
+
+class DatabaseConfigSerializer implements IDatabaseConfigSerializer
```

Comment:
Neviem ci to niekde okrem `oracle` pouzijeme, ... pridal by som nad tieto nové `*Serializer` triedy koment naco to je dobre .... (v `GetTableMetadataSerializer` a `ManifestMetadataSerializer` je pouzitie jasne)

Napr. `This class can be used if you need to pass parameters to a external tool (eg. java-oracle-exporter).`

Bez toho komentara je podla mna tazke prist na to, na co to je.

Reply:
jo jasně, hned to tam bude